go/vt/vttest/local_cluster.go

```diff
@@ -185,7 +185,7 @@ type LocalCluster struct {
 // This connection should be used for debug/introspection purposes; normal
 // cluster access should be performed through the vtgate port.
 func (db *LocalCluster) MySQLConnParams() mysql.ConnParams {
-	return db.mysql.Params(db.DbName())
+	return db.mysql.Params("")
```

Comment:
Why this change? This might be causing some of the test failures.

Reply:
I see now. Pushing a more correct fix to LocalCluster:

Reply:
```

Reply:
```
The DbName function for the local cluster was returning an
incorrect name previously. It was the keyspace name, which would
not match the db names in mysql. The previous fix stopped using
DbName for setting the mysql parameters, which broke some tests.

The new behavior fixes DbName to return a value only if it was
in OnlyMySQL mode.
```